foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/java/JavaStep.java

```diff
@@ -453,6 +453,12 @@ public String getLowerCamelCaseName() {
     }
 
     private void addRecordImports(String importName) {
+        if(importName == null)
+        {
+            throw new GenerationException("Step '" + getName() + "' has outbound or inbound Record Type that have "
+                    + "null import name.");
+        }
+
```

Comment:
I: this is to have better error thrown than plain NPE

Reply:
Q: Did you see this happen in testing? I would have expected the model validation phase to throw an error before we got to this point.  Not bad to have this either way, but if we're missing something in validation it'd be good to fill that gap.